atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -106,6 +109,11 @@ protected int getPoolSize() {
             if (resource != null) {
                 if (shouldReuse) {
                     log.debug("Returning {} to pool", resource);
+                    TTransport transport = resource.getInputProtocol().getTransport();
+                    if (transport instanceof TMemoryInputTransport) {
+                        log.debug("During {} check-in, cleaned up an iprot of size {}", resource, transport.getBuffer().length);
```

Comment:
`getBuffer()` can return null (though possibly not in this case) and then we'd NPE.

Do we care about any other transport types?